### PR TITLE
improve docs - raw count is not the only indicator

### PR DESCRIPTION
**Pull request by matkoniecz**
triggered by #2461

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -259,10 +259,11 @@ and a `tourism=motel` (correct). In this situation we want to:
 * Delete the entry for `"tourism/hotel|Motel 6"` and
 * Add `"match": ["tourism/hotel|Motel 6"]` to the `"tourism/motel|Motel 6"` entry
 
-Usually the entry used more frequently (indicated by "count" property)
-is the one to keep.  If you are not sure, you can also search on the
-[OpenStreetMap Wiki](https://wiki.openstreetmap.org/wiki/Map_Features) for tag best practices.
+Local knowledge, existing tagging (indicated by "count" property), information at Wikipedia page or company's website, [OpenStreetMap Wiki tag documentation](https://wiki.openstreetmap.org/wiki/Map_Features) help in deciding which entry should be kept.
```
Comment (waldyrious) [inline CONTRIBUTING.md]:
```suggestion
Local knowledge, existing tagging (indicated by the `count` property), information at the relevant Wikipedia page or the company's website, and [OpenStreetMap Wiki tag documentation](https://wiki.openstreetmap.org/wiki/Map_Features) all help in deciding which entry should be kept.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -259,10 +259,11 @@ and a `tourism=motel` (correct). In this situation we want to:
...
 
+If situation is unclear one may contact [local community](https://community.osm.be/) and ask for help.
```
Comment (waldyrious) [inline CONTRIBUTING.md]:
```suggestion
If the situation is unclear, one may contact the [local community](https://community.osm.be/) and ask for help.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -259,10 +259,11 @@ and a `tourism=motel` (correct). In this situation we want to:
...
+
+Note that in some cases both entries should be kept - for example given brand may really operate both superkarkets and convenience stores under the same name. In that case it is necessary to use `nomatch`.
```
Comment (waldyrious) [inline CONTRIBUTING.md]:
```suggestion
Note that in some cases both entries should be kept - for example, a given brand may really operate both superkarkets and convenience stores under the same name. In that case it is necessary to use the `nomatch` property.
```

**Comment by matkoniecz** [comment]
Thanks for fixes!

**Pull request merged**